beacon-chain/core/helpers/committee.go

```diff
@@ -293,41 +304,45 @@ func AttestationParticipants(
 			participants = append(participants, validatorIndex)
 		}
 	}
-	sort.Slice(participants, func(i, j int) bool { return participants[i] < participants[j] })
 	return participants, nil
 }
 
-// VerifyBitfield verifies bitfield against the committee_size.
+// VerifyBitfield validates a bitfield with a given committee size.
 //
-// Spec pseudocode definition:
-//   def verify_bitfield(bitfield: bytes, committee_size: int) -> bool:
-//     """
-//     Verify ``bitfield`` against the ``committee_size``.
-//     """
-//     if len(bitfield) != (committee_size + 7) // 8:
-//         return False
-//     # Check `bitfield` is padded with zero bits only
-//     for i in range(committee_size, len(bitfield) * 8):
-//         if get_bitfield_bit(bitfield, i) == 0b1:
-//             return False
-//     return True
+// Spec pseudocode:
+//
+// def verify_bitfield(bitfield: bytes, committee_size: int) -> bool:
+// """
+// Verify ``bitfield`` against the ``committee_size``.
+// """
+// if len(bitfield) != (committee_size + 7) // 8:
+// return False
+//
+// # Check `bitfield` is padded with zero bits only
+// for i in range(committee_size, len(bitfield) * 8):
+// if get_bitfield_bit(bitfield, i) == 0b1:
+// return False
+//
+// return True
 func VerifyBitfield(bitfield []byte, committeeSize int) (bool, error) {
 	if len(bitfield) != mathutil.CeilDiv8(committeeSize) {
 		return false, fmt.Errorf(
 			"wanted participants bitfield length %d, got: %d",
 			mathutil.CeilDiv8(committeeSize),
 			len(bitfield))
 	}
-	bitLength := len(bitfield) << 3
-	for i := committeeSize; i < bitLength; i++ {
-		set, err := bitutil.CheckBit(bitfield, i)
+
```

Comment:
why is this changed ?

Reply:
I copied it get_epoch_committee_count implementation and tests from #2354
and copied get_epoch_start_shard implementation and test from #2497